### PR TITLE
Taking out  bayesian_topological_localisation

### PR DESCRIPTION
**Pull request by francescodelduchetto**
The localisation package is now at https://github.com/LCAS/bayesian_topological_localisation

### CODE REVIEW

**Comment by Iranaphor** [comment]
Were there any files in topological_navigation_msgs which should be isolated along with this?

**Comment by francescodelduchetto** [comment]
> Were there any files in topological_navigation_msgs which should be isolated along with this?

I believe not. Messages and services definitions are https://github.com/LCAS/topological_navigation/tree/master/bayesian_topological_localisation/msg and https://github.com/LCAS/topological_navigation/tree/master/bayesian_topological_localisation/srv

**Comment by francescodelduchetto** [comment]
> > Were there any files in topological_navigation_msgs which should be isolated along with this?
> 
> I believe not. Messages and services definitions are https://github.com/LCAS/topological_navigation/tree/master/bayesian_topological_localisation/msg and https://github.com/LCAS/topological_navigation/tree/master/bayesian_topological_localisation/srv

Ah no actually, you are right....with the move to humble also these definitions have been moved it seems...

**Comment by francescodelduchetto** [comment]
@Iranaphor, please check it's all correct now.

**Comment by Iranaphor** [comment]
That was my only thought, with the humble transition I figured tighter integration would be beneficial. Im happy.

**Comment by marc-hanheide** [comment]
still test fails as messages are referenced that are no longer there, right?

**Comment by marc-hanheide** [comment]
see https://github.com/LCAS/topological_navigation/actions/runs/7505325308/job/20441251372#step:5:179

**Comment by marc-hanheide** [comment]
removed the building of the no longer existing messages

File: topological_navigation_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -35,11 +35,7 @@ set(msg_deps
 
 # Declare the custom message files
 set(msg_files
-  "msg/DistributionStamped.msg"
   "msg/ExecutePolicyModeGoal.msg"
-  "msg/PoseObservation.msg"
-  "msg/LikelihoodObservation.msg"
-  "msg/ParticlesState.msg"
   "msg/ClosestEdges.msg"
```
Comment (marc-hanheide) [inline topological_navigation_msgs/CMakeLists.txt:39]: With these changes, I think this is now good to be merged

**Review by marc-hanheide — APPROVED** [review]
happy now

**Comment by marc-hanheide** [comment]
@Iranaphor you also happy? If so, let's merge.

**Pull request merged**